packages/apps/transfer/src/pages/faucet/existing/index.tsx

```diff
@@ -0,0 +1,189 @@
+import { IPollResponse } from '@kadena/chainweb-node-client';
+import { ICommandBuilder, IPactCommand, PactCommand } from '@kadena/client';
+import {
+  Button,
+  Heading,
+  SystemIcons,
+  TextField,
+} from '@kadena/react-components';
+
+import FormStatusNotification from './notification';
+
+import MainLayout from '@/components/Common/Layout/MainLayout';
+import { Select } from '@/components/Global';
+import { StyledOption } from '@/components/Global/Select/styles';
+import {
+  StyledAccountForm,
+  StyledForm,
+  StyledFormButton,
+} from '@/pages/transfer/cross-chain-transfer-finisher/styles';
+import { fundExistingAccount } from '@/services/faucet';
+import useTranslation from 'next-translate/useTranslation';
+import React, {
+  FC,
+  FormEvent,
+  FormEventHandler,
+  useCallback,
+  useState,
+} from 'react';
+
+// TODO: This needs to be changed to 100, when the contract is redeployed
+const AMOUNT_OF_COINS_FUNDED: number = 20;
+
+// eslint-disable-next-line @kadena-dev/typedef-var
+export const CHAINS = [
```

Comment:
are chains reused? if yes they can be moved to `@/types/xxx`  in a new file maybe!

Reply:
Not right now, I do see a chain select component in other pages as well, I will refactor this when working on the refactor ticket 👍